src/app.js

```diff
@@ -1 +1,27 @@
 'use strict';
+
+const { generateNumber } = require('./numGenerator');
+const { bullsAndCows } = require('./bullsAndCows');
+const { inputFunc } = require('./inputOutput');
```

Comment:
check function name, let's not make it shortened as it affects in bad way on the code readability